tests/pipelines/test_pipelines_automatic_speech_recognition.py

```diff
@@ -377,6 +379,49 @@ def test_chunking_fast(self):
         self.assertEqual(output, [{"text": ANY(str)}])
         self.assertEqual(output[0]["text"][:6], "ZBT ZC")
 
+    @require_torch
+    @slow
+    def test_mctct(self):
+        speech_recognizer = pipeline(
+            model="speechbrain/m-ctc-t-large",
+            chunk_length_s=10.0,
+            return_timestamps="word",
+            framework="pt",
+        )
+
+        ds = load_dataset("hf-internal-testing/librispeech_asr_dummy", "clean", split="validation").sort("id")
+        audio = ds[40]["audio"]["array"]
+
+        # It should work for a single non-batched input
+        output = speech_recognizer(audio)
+        self.assertEqual(
+            nested_simplify(output),
+            {
+                "text": "A man said to the universe, Sir, I exist.",
+                "chunks": [
+                    {"text": "A", "timestamp": (0.3, 0.33)},
+                    {"text": "man", "timestamp": (0.51, 0.72)},
+                    {"text": "said", "timestamp": (0.93, 1.14)},
+                    {"text": "to", "timestamp": (1.2, 1.29)},
+                    {"text": "the", "timestamp": (1.38, 1.47)},
+                    {"text": "universe,", "timestamp": (1.59, 2.4)},
+                    {"text": "Sir,", "timestamp": (2.67, 3.15)},
+                    {"text": "I", "timestamp": (3.33, 3.36)},
+                    {"text": "exist.", "timestamp": (3.6, 4.59)},
```

Comment:
Those offsets are roughly correct, and didn't match in the original PR.

Reply:
Because this is a trained big model, we can test ALL the outputs.

This wasn't true in the original test it was copied from, because random models that work letter per letter was outputting a LOT of dummy output, so we had to slice it somehow to keep the test readable. (We could also have chunked the input audio,  but under a certain size, all the output was "", and suddenly triggered a ton of output after a certain length).